ServiceApp/src/org/droidplanner/services/android/core/mission/survey/SurveyImpl.java

```diff
@@ -115,6 +119,26 @@ public static msg_mission_item packSurveyPoint(LatLong point, double altitude) {
         return mavMsg;
     }
 
+    private msg_mission_item getYawCondition(double angle){
+        return packYawCondition(angle);
+    }
+
+    public static msg_mission_item packYawCondition(double angle){
```

Comment:
@TShapinsky this method is only used by `getYawCondition(...)`, so let's just move its logic within `getYawCondition(...)`, unless you're planning to use it somewhere else.